api/src/main/java/io/strimzi/api/kafka/model/status/ListenerStatus.java

```diff
@@ -49,6 +50,15 @@ public void setType(String type) {
         this.type = type;
     }
 
+    @Description("The `name` field reflects the same value as the `type` field. Once the `type` field gets removed, this will reflect the actual name of the listener.")
```

Comment:
You should explain what it is and not what it replaces or what has the same value. Just say that this is the name of the listener.

Reply:
```suggestion
    @Description("The name of the listener.")
```

@ShubhamRwt the reason to do it like this is it make the eventual removal PR much simpler if you only have to remove the getter and setter for type. If you also have to go changing the docs for name as well (so they no longer refer to the now removed `type`) then it's more work.